### PR TITLE
Update morty release

### PR DESCRIPTION
**Pull request by bavery22**

### CODE REVIEW

**Review by rewitt1 — COMMENTED** [review]
Ideally this should be two commits because the changes aren't related to each other.

**Review by rewitt1 — COMMENTED** [review]
Really I would just specify the ones it is needed FOR.

Also maybe an explanation of why the heck it has to persist, i.e what it's solving.

**Comment by bavery22** [comment]
info added/removed. force pushed.

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +26,7 @@ env:
     # LATEST_RELEASE_REPO is the dockerhub repo that FLOATING_REPO should
```
Comment (rewitt1) [inline .travis.yml:26]: Should really change the FLOATING_REPO comment to match, otherwise it could be a bit confusing.

File: .travis.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@ env:
     - REPO=crops/toaster-krogoth BRANCH=krogoth DOCKERHUB_TAG="latest" POKYBRANCH=krogoth
     - REPO=crops/toaster-krogoth BRANCH=yocto-2.1 DOCKERHUB_TAG="2.1" POKYBRANCH=krogoth
     - REPO=crops/toaster-krogoth BRANCH=yocto-2.1.1 DOCKERHUB_TAG="2.1.1" POKYBRANCH=krogoth
+    - REPO=crops/toaster-krogoth BRANCH=yocto-2.1.2 DOCKERHUB_TAG="2.1.2" POKYBRANCH=krogoth
```
Comment (rewitt1) [inline .travis.yml:18]: I put it in the other review as well, but this should be a separate commit, because it doesn't have anything to do with updating to morty.
